Projekty/Projekt2/Marcin Wilk, Hubert Drążkowski/[Grupa_4][Milestone_1]WilkDrazkowski.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<p>Tutaj padło pytanie czy dodawanie takich cech to dobra droga. Moim zdaniem jak najbardziej. Można też spróbować jak klasteryzacja będzie działać bezpośrednio na Bag Of Words (tutaj pomysł z wyrzuceniem rzadkich słów jest dobry). </p><p>W przypadku oceny sentymentu lepiej pracować na oryginalnych danych. Może warto wyrzucić znaki specjalne typu \n, paragraf, itp. </p><p>Podsumowując proponuje dwa podejścia do klasteryzacji z wykorzystaniem:</p><ul><li>Bag Of Words</li><li>samodzielnie stworzonych cech np.: liczba wyrazów, liczba zdań, liczba unikalnych słów, subjectivity, polarity itp</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/317/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>